lib/logging/summary.js

```diff
@@ -17,6 +18,15 @@ module.exports = function summary(request, response) {
     scope: request.auth.credentials.scope
   };
 
+  var payloadKeys = [];
+  try {
+    // given payload object might not be a valid object
+    // See issue #410
+    payloadKeys = Object.keys(payload);
+  } catch (e) {
+    // failed to parse payload keys.
```

Comment:
Small suggestion. Factor the try/catch into a helper function. Otherwise v8 will not optimize the exported function.